build/js/movie_details.js

```diff
@@ -9,11 +9,11 @@ const movie_details = document.querySelector('.movie_details');
 // const movieID = localStorage.getItem('id');
 const image_baseurl = 'https://image.tmdb.org/t/p/';
 
-document.addEventListener('click', e => {
-  if (e.target && e.target.id === 'goback') {
-    window.location.href = './index.html';
-  }
-});
+// document.addEventListener('click', e => {
+//   if (e.target && e.target.id === 'goback') {
+//     window.location.href = './index.html';
+//   }
+// });
```

Comment:
Can this be deleted since it's no longer used?

Reply:
Yes it can. I need to clean that code up for sure.